content/app/development/data/prefill/_index.en.md

```diff
@@ -2,10 +2,14 @@
 title: Prefilling data
 linktitle: Prefill
 description: How to configure prefill for an app.
-toc: true
+toc: false
 weight: 200
 ---
 
+{{% version-selector %}}
+
+{{< version id="1.x 2.x" >}}
```

Comment:
Space is not valid in id-attributes in HTML.

Reply:
No longer using space in id-attributes.